test/validators.js

```diff
@@ -3574,6 +3565,17 @@ describe('Validators', function () {
           '0 1234 5689',
           '16 1234 5689',
           '03_1234_5689',
+          '0312345678',
+          '0721234567',
+          '08002345678'
```

Comment:
You're missing a trailing comma here. See the [test log](https://travis-ci.org/chriso/validator.js/jobs/291646468).